multiview/cotraining/base.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This is a base class for implementing multi-view estimators with
+# co-training.
+
+from abc import abstractmethod
+import numpy as np
+from sklearn.base import BaseEstimator
+
+
+class BaseCoTrainEstimator(BaseEstimator):
+    """
+    A base class for multiview co-training.
+    Parameters
+    ----------
+    BaseEstimator : Estimator abstract class
```

Comment:
I fked up sorry lol. Just assumed BaseEstimator was a parameter by accident. Can you instead just add a line under "a base class for multiview co-training" that we extend from sklearn's base estimator class?

Reply:
done